src/agent/onefuzz-supervisor/src/worker/tests.rs

```diff
@@ -64,7 +64,9 @@ impl IWorkerRunner for RunnerDouble {
 async fn test_ready_run() {
     let mut runner = Fixture.runner(Fixture.child_running());
     let state = State {
-        ctx: Ready,
+        ctx: Ready {
+            setup_dir: PathBuf::from(""),
```

Comment:
FYI, this can be achieved by either `PathBuf::new()` and `PathBuf::default()`.

That said, when testing, I suggest we use some fake but nonempty value.